morango/sync/syncsession.py

```diff
@@ -423,25 +528,19 @@ def initiate_push(self, sync_filter):
         self._close_transfer_session()
 
     def initiate_pull(self, sync_filter):
-
-        logger.info("Initiating pull sync")
         self._create_transfer_session(False, sync_filter)
 
         if self.current_transfer_session.records_total == 0:
-            logger.info("There are no records to transfer")
```

Comment:
This may still be a useful logger output to keep.

Reply:
But may make sense to shift this block (lines 533-535) down into the signal context (before line 540), so that the signals would still be fired (even if in rapid succession), in case something external was expecting them.

Reply:
I rearranged the code a little bit and will make sure we're report this information specifically in Kolibri.